api/controllers/apps_test.go

```diff
@@ -85,6 +94,20 @@ func TestAppCreateWithAlreadyExists(t *testing.T) {
 	test.AssertStatus(t, 403, "POST", "http://convox/apps", val)
 }
 
+func TestAppCreateWithAlreadyExistsBound(t *testing.T) {
+	aws := test.StubAws(
+		test.DescribeStackNotFound("application"),
+		test.DescribeAppStackCycle("convox-test-application"),
+		test.CreateAppStackExistsCycle("convox-test-application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeAppStackCycle("convox-test-application"),
```

Comment:
Once `AppCreate` uses `DescribeStacks` directly, and the un/bound order is swapped, this *should* fall out.